trade-finance-manager-ui/templates/utilisation-reports/add-payment.njk

```diff
@@ -0,0 +1,17 @@
+{% import "./_macros/selected-reported-fees-details-table.njk" as selectedReportedFeesDetailsTable %}
+
+{% extends "index.njk" %}
+
+{% block pageTitle %}Add a payment{% endblock%}
+{% block content %}
+  <div class="govuk-grid-row govuk-!-padding-top-7">
+    <div class="govuk-grid-column-one-half">
+      <span class="govuk-caption-l" data-cy="add-a-payment-heading-caption">{{ bank.name }}, {{ reportPeriod }}</span>
```

Comment:
Could we maybe call `reportPeriod` `formattedReportPeriod` to be consistent with the other templates?